modules/nf-core/dysgu/main.nf

```diff
@@ -0,0 +1,60 @@
+
+process DYSGU {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/dysgu:latest' :
+        'biocontainers/dysgu:latest' }"
+
+    input:
+
+    tuple val(meta), path(input_bam), path(input_bam_index)
+    tuple val(meta2), path(reference_fasta), path(reference_fasta.fai)
+    tuple val(meta3), path(temp_dir)
+
+    output:
+
+    tuple val(meta), path('*.vcf.gz')        , emit: vcf
+    tuple val(meta), path('*.vcf.gz.tbi')   , emit: tbi
+
+    path 'versions.yml'           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def args3 = task.ext.args3 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    dysgu run \\
+        $reference \\
+        $temp_dir \\
+        $input_bam \\
+       | bgzip ${args2} --threads ${task.cpus} --stdout > ${prefix}.vcf.gz
+     tabix ${args3} ${prefix}.vcf.gz
```

Comment:
That were probably my changes which introduced these errors:
```suggestion
        | bgzip ${args2} --threads ${task.cpus} --stdout > ${prefix}.vcf.gz
    tabix ${args3} ${prefix}.vcf.gz
```
Maybe try these.